### ISSUE TITLE
A22ly concerns 

### ISSUE THREAD

**Issue by hosein2398** [hosein2398/react-textra#2]
Should make this accessible, this can be done by adding props like:
```
aria-live="polite"
aria-relevant="text removals"
```

Have to decide to make it optional in the library or not.

**Comment by geekyarjun** [comment]
I think it should be in the library as optional. It is indeed helpful for a11y and providing a way to opt-in a11y would be a big plus.
